Gemfile

```diff
@@ -13,6 +13,8 @@ gem 'active_hash'
 gem 'sanitize'
 gem 'gmaps4rails'
 gem 'geocoder'
+gem 'omniauth-google-oauth2'
+gem 'omniauth-oauth2', '~> 1.3.1'
```

Comment:
@tjgrathwell I could've sworn that I was running into that error that you pointed me to, and that locking `omniauth-oauth2` fixed it. I don't get the error now, after following your directions. A bit perplexed about it, I have to admit.